src/styles/settings.scss

```diff
@@ -26,3 +26,7 @@
   text-decoration: none;
 }
 
+p {
```

Comment:
oof what was reasoning behind this? never a big fan of these glbal styles as its hard for the next person to debug why their p tag margins are weird

Reply:
I'm adding 3 static pages to the site with similar styling, so it would be helpful to define p and ul globally so I don't have to duplicate those styles across 3 templates.

Reply:
That said, I'm not sure which p tags this change is targeted at; if it's something on the map as opposed to the about page then I'd prefer these to have a more specific selector.

Reply:
Yeah, it should be a consistent style for the non-map pages. Will add to a class.